source/jormungandr/jormungandr/georef.py

```diff
@@ -62,6 +62,28 @@ def place(self, place):
         response = self.instance.send_and_receive(req)
         return response.places[0]
 
+    def direct_path(self, mode, origin, destination, datetime, clockwise):
+        logger = logging.getLogger(__name__)
+        req = request_pb2.Request()
+        req.requested_api = type_pb2.direct_path
+        req.direct_path.origin.place = origin
+        req.direct_path.origin.access_duration = 0
+        req.direct_path.destination.place = destination
+        req.direct_path.destination.access_duration = 0
+        req.direct_path.datetime = datetime
+        req.direct_path.clockwise = clockwise
+        req.direct_path.streetnetwork_params.origin_mode = mode
+        req.direct_path.streetnetwork_params.destination_mode = mode
+        req.direct_path.streetnetwork_params.walking_speed = self.instance.walking_speed
+        req.direct_path.streetnetwork_params.max_walking_duration_to_pt = self.instance.max_walking_duration_to_pt
```

Comment:
In the code we don't do x2, we do origin_max_duration + destination_max_duration. But I can do that.